From: Jan Engelhardt <jengelh@medozas.de>
Upstream: never

Replace __DATE__ and __TIME__ with static text.

Note: rpmlint will still complain when the static date that was
encoded happens to be the current day. That is ok, since rpmlint
uses just a heuristic - excessive rebuilds should not actually
be happening.
---
 schism/version.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: schismtracker-20241226/schism/version.c
===================================================================
--- schismtracker-20241226.orig/schism/version.c
+++ schismtracker-20241226/schism/version.c
@@ -53,7 +53,7 @@ static const char* top_banner_normal =
 #ifndef EMPTY_VERSION
 	"Schism Tracker " VERSION
 #else
-	"Schism Tracker built " __DATE__ " " __TIME__
+	"Schism Tracker built <>"
 #endif
 	;
 
